PocketKit/Sources/PocketKit/Settings/AccountViewModel.swift

```diff
@@ -18,6 +26,23 @@ class AccountViewModel: ObservableObject {
         appSession.currentSession = nil
     }
 
+    func toggleAppBadge() {
+        UNUserNotificationCenter.current().requestAuthorization(options: .badge) {
+            (granted, error) in
+            guard error == nil && granted == true else {
+                self.userDefaults.set(false, forKey: "SettingsRowToggle")
```

Comment:
Is the key here supposed to be different than the one used by `AppStorage`?

EDIT: I think I know what's going on, but I'll need to verify…

Reply:
nope your right. oversight on my part.